Scripts/Core/Contexts/EditingContextManager.cs

```diff
@@ -0,0 +1,282 @@
+﻿#if UNITY_EDITOR && UNITY_EDITORVR
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor.Experimental.EditorVR.Utilities;
+using UnityEngine;
+using UnityEngine.Assertions;
+using UnityEngine.InputNew;
+using System.IO;
+
+namespace UnityEditor.Experimental.EditorVR.Core
+{
+	class EditingContextManager : MonoBehaviour
+	{
+		const string k_SettingsPath = "ProjectSettings/EditingContextManagerSettings.asset";
+		const string k_UserSettingsPath = "Library/EditingContextManagerSettings.asset";
+
+		static EditingContextManager s_Instance;
+		static InputManager s_InputManager;
+
+		EditingContextManagerSettings m_Settings;
+
+		List<IEditingContext> m_AvailableContexts;
+		string[] m_ContextNames;
+		int m_SelectedContextIndex;
+
+		IEditingContext m_CurrentContext;
+		readonly List<IEditingContext> m_PreviousContexts = new List<IEditingContext>();
+
+		IEditingContext defaultContext
+		{
+			get
+			{
+				var context = m_AvailableContexts.First();
```

Comment:
Without having picked a default, I'll be loaded into the blink context. Without somehow hardcoding EditorVR to set itself up as a default, I would like to preserve the "first run experience" of starting in the EditorVR context.

I see an interesting opportunity here: Create a _Tutorial context that we can count on to be sorted first, which introduces the user to  EVR and as a part of the tutorial sets up EVR as the default context. In any case, we could just rename Blink to OnlyBlink and rely on the fact that files sort alphabetically and OnlyBlink comes after EditorVR.

Otherwise, I wonder if there is a more clever way to default to EditorVR when you first set up the project.